workspaces/tech-insights/plugins/tech-insights-maturity/report.api.md

```diff
@@ -42,9 +42,6 @@ export type MaturityApi = TechInsightsApi & {
   getBulkMaturityCheckResults(
     entities: Entity[],
   ): Promise<BulkMaturityCheckResponse>;
-  getChildMaturityCheckResults(
```

Comment:
Should this be a major change if we are removing a function from this api?

If it is public, how do we know it is unused?

Reply:
Yeah I think its probably best to set it as deprecated. Can I ask why its necessary to remove it?

Reply:
sorry just got back from a long vacation. It's not necessary to deprecate or remove it so I added it back